src/Winp/Form/ServiceForm.cs

```diff
@@ -84,7 +85,14 @@ public ServiceForm()
         _packagePhpStatusLabel.ImageList = statusImageList;
         _packagePhpMyAdminStatusLabel.ImageList = statusImageList;
         _services = [mariaDbService, nginxService, phpService, phpMyAdminService];
-        _scheduler = scheduler;
+
+        // Configure NotifyIcon (it's initialized in Designer.cs)
+        if (_notifyIcon != null)
+        {
+            _notifyIcon.Icon = this.Icon ?? SystemIcons.Application;

```

Comment:
Still no "this." please 🙏

Why do you need the fallback here? Isn't replicating current form's one OK?